### PR TITLE
fix shell vs bash issue

### PR DESCRIPTION
**Pull request by deepanshu-yadav**

### CODE REVIEW

File: extras_install.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ echo "Platform is ${os_name}, Version: ${os_version}"
 ID=$(cat /etc/os-release | grep -w ID | cut -f2 -d"=")
 
 if [ ${ID} != "mendel" ]; then
-  if [[  $os_name == *"Red Hat"* || $os_name == *"CentOS"*  ]]; then
+  if [[ "$os_name" == *"Red Hat"* ]] || [[ "$os_name" == *"CentOS"* ]]; then
```
Comment (praveen-garg) [inline extras_install.sh:10]: ` if [[ "$os_name" == *"Red Hat"*  ||  "$os_name" == *"CentOS"* ]]; then`  should work fine.

Comment (deepanshu-yadav) [reply extras_install.sh:10]: not working

Comment (praveen-garg) [reply extras_install.sh]: OK, go ahead with whatever works!

File: requirements.sh [hunk 2]
```diff
@@ -22,7 +22,7 @@ os_name=$(grep -o '^NAME=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')
 os_version=$(grep -o '^VERSION_ID=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')
 echo "Platform is ${os_name}, Version: ${os_version}"
 
-if [[  $os_name == *"Red Hat"* || $os_name == *"CentOS"*  ]]; then
+if [[ "$os_name" == *"Red Hat"* ]] || [[ "$os_name" == *"CentOS"* ]]; then
```
Comment (praveen-garg) [inline requirements.sh:25]: Should this also be made sh compatible?

Comment (deepanshu-yadav) [reply requirements.sh]: Converted to a case statement. Now it is working.

**Review by YashTatkondawar — APPROVED** [review]

**Pull request merged**